client/src/pages/Contacts/index.js

```diff
@@ -1,4 +1,173 @@
+    try {
+      e.preventDefault();
+      setLoading(true);
+      const contactInfo = { username, email, message, mobile };
+      await contactValidationSchema.isValid(contactInfo, { abortEarly: false });
+      await Axios.post('/api/v1/contact-us', contactInfo);
+      clear();
+      setLoading(false);
+      setOpen(true);
+    } catch (err) {
+      setError(
+        err.response.data.message
+          ? err.response.data.message
+          : 'Internal server error'
+      );
+      setLoading(false);
+    }
+  };
+
+  return (
+    <>
+      <Header />
+      <Container className={classes.root}>
+        <Typography> The Beauty - Gaza Beauty</Typography>
+        <Divider component="hr" variant="middle" className={classes.divider} />
+        <Typography>
+          Villa 1043, Al Wasl Road, Umm Suqeim 2, Near J3 Mall, palestine Gaza
+          mobile: 04 333 7772
+        </Typography>
+        <Divider component="hr" variant="middle" className={classes.divider} />
+        <img src={map} alt="map" className={classes.img} />
+        <Typography className={classes.title} component="h3" variant="h5">
+          Contact us
+        </Typography>
+        <form className={classes.form}>
+          <Container className={classes.container}>
+            <InputField
+              label="Name *"
+              value={username}
+              onChange={handelChange}
+              name="username"
+              margin="dense"
+            />
+            <InputField
+              label="mobile *"
+              name="mobile"
+              value={mobile}
+              onChange={handelChange}
+              margin="dense"
+            />
+            <InputField
+              label="Email *"
+              name="email"
+              type="email"
+              value={email}
+              onChange={handelChange}
+              margin="dense"
+            />
+            {loading && <Loading size={40} color="primary" />}
+            {isMobile && (
+              <TextareaAutosize
+                rowsMin={5}
+                value={message}
+                name="message"
+                placeholder="Message *"
+                onChange={handelChange}
+                className={classes.message}
+              />
+            )}
+            <ButtonComponent onClick={handleSubmit} className={classes.button}>
+              Submit
+            </ButtonComponent>
+            <Snackbar open={open} autoHideDuration={3000} onClose={handleClose}>
+              <Alert onClose={handleClose} severity="success">
+                Message sent successfully
+              </Alert>
+            </Snackbar>
+          </Container>
+          {isMobile ? (
+            ''
+          ) : (
+            <TextareaAutosize
+              rowsMin={9}
+              value={message}
+              name="message"
+              placeholder="Message *"
+              onChange={handelChange}
+              className={classes.message}
+            />
```

Comment:
suggestion: you can use material UI input field as text area to make its style like other input fields in contact us form
```suggestion
          <InputField
              label="message"
              rows={5}
              multiline
              value={message}
              name="message"
              onChange={handelChange}
            />
```